src/ripple/ledger/View.h

```diff
@@ -478,7 +478,7 @@ using EntryDeleter = std::function<std::pair<TER, SkipEntry>(
 cleanupOnAccountDelete(
     ApplyView& view,
     Keylet const& ownerDirKeylet,
-    EntryDeleter deleter,
+    EntryDeleter const& deleter,
```

Comment:
I probably wouldn't have made this `const`. If the callback wants to change itself (increment some counter say), the interface shouldn't stop that.

Reply:
It would not compile otherwise.